docs/content/manual/_incoming/3045-upgrade-with-bonded-nic.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Upgrade Harvester on node that has bonded NICs for management interface
+---
+
+* Related issues: [#3045](https://github.com/harvester/harvester/issues/3045) [BUG] Harvester Upgrade 1.0.3 to 1.1.0 does not handle multiple SLAVE in BOND for management interface
+
+## Category: 
+* Upgrade
+
+## Environment Setup
+
+- This is to be done on a Harvester cluster where the NICs were configured to be bonded on install for the management interface
```

Comment:
It will be good if we add other env details like how many node etc.

Reply:
@khushboo-rancher - I think right now this would have to be restricted to purely virtualized testing (at least for now :sweat_smile: ), since we do not have a bare metal environment that has two 10GiB NIC cards present in it. - Though I think it could be done on either "if" a bare metal environment was set up with 2 10GiB NIC cards in the node.  

I think single-node would be fine since the management network is restricted to the "create" node.  And since currently after the creation of the mgmt based network we don't allow for it to be modified from the UI directly - to add additional network card ports, x-ref:
![Screenshot from 2022-11-02 10-54-31](https://user-images.githubusercontent.com/5370752/199565291-1573cc40-885a-4068-a80d-e722023b2f60.png)